### PR TITLE
Fixed the subject urls from for example <dockevent:..>

### PR DESCRIPTION
**Pull request by fr0gs**
- Added a new class to represent the **DockEventActor** 
- Fixed the prefixes from <dockevent: etc..> to expand the prefix.

Does not break any other microservices while testing as a whole so far.

### CODE REVIEW

File: muswarmlogger/loggers/sparql/events.py [hunk 1]
```diff
@@ -29,8 +29,8 @@
     _datetime = datetime.fromtimestamp(int(_time))
 
     event_id = "%s_%s" % (event_id, _timeNano)
-    event_node = Node("<dockevent:%s>" % event_id, {
-        "a": DockEventTypes.event,
+    event_node = Node(DockEvent.__iri__+ "%s" % event_id, {
```
Comment (cecton) [inline muswarmlogger/loggers/sparql/events.py]:
```
+ event_id
```

should work

**Review by cecton — CHANGES_REQUESTED** [review]
Small change but fine

**Review by cecton — APPROVED** [review]
@fr0gs to me that's fine but I just realized now that you're changing the URLs of the subjects. Maybe check with @earthquakesan that this is correct I don't know.

**Comment by fr0gs** [comment]
I will wait until @earthquakesan gives an answer then. We have been using subjects with a wrong url until this was fixed so since it does not break the integrator it should be ready.

**Comment by cecton** [comment]
No but I think we have documentation for the vocabulary and stuff, I don't know if the URLs are taken into account... Maybe we got the doc out-of-sync again for some time.

https://github.com/big-data-europe/mu-swarm-logger-service/blob/master/docs/docker-engine-events/events.owl

**Pull request merged**